irrd/webui/endpoints.py

```diff
@@ -172,3 +179,60 @@ def save():
             },
         )
     return Response(status_code=405)  # pragma: no cover
+
+
+@session_provider_manager
+@authentication_required
+async def change_log_mntner(request: Request, session_provider: ORMSessionProvider) -> Response:
+    query = session_provider.session.query(AuthMntner).join(AuthPermission)
+    query = query.filter(
+        AuthMntner.pk == request.path_params["mntner"],
+        AuthPermission.user_id == str(request.auth.user.pk),
```

Comment:
verify that this performs the join correctly, extend tests for this case (tests now only do no AuthPermission at all)